v8/client/network.go

```diff
@@ -109,6 +109,10 @@ func dialSendUDP(kdcs map[int]string, b []byte) ([]byte, error) {
 		}
 		return rb, nil
 	}
+	if len(errs) > 0 {
+		errorString := strings.Join(errs, ",")
+		return nil, fmt.Errorf("error in getting a UDP connection to any of the KDCs: %v", errorString)
+	}
 	return nil, fmt.Errorf("error sending to a KDC: %s", strings.Join(errs, "; "))
```

Comment:
Sorry for not spotting this earlier. Seems like the dialSendUdp() function was already handling this. Perhaps the best fix is to adopt this style of handling it in the TCP equivalent.